src/module/Provider.sol

```diff
@@ -72,6 +75,19 @@ contract Provider is IProvider {
             return IsfrxETH(MC.SFRXETH).pricePerShare() * frxETHPriceInETH / 1e18;
         }
 
+        if (asset == MC.CURVE_LP_YNETH_YNLSDE_STRATEGY) {
+            (int256 lpRate, uint256 updatedAt) = ICurveLpConnector(MC.CURVE_LP_YNETH_YNLSDE_CONNECTOR).rate();
+            if (lpRate < 0) {
+                revert RateIsNegative();
+            }
+            // solhint-disable-next-line not-rely-on-time
+            if (updatedAt < block.timestamp - 24 hours) {
```

Comment:
also this check, do you think its important to have?

Reply:
no it's fine let's leave it out currently it doesn't get stale

good point though

Reply:
removed in [742159e](https://github.com/yieldnest/yieldnest-vault/pull/88/commits/742159e4f34ba39e9ca27f30c84a4de4edfcf847)